source/Irrlicht/COpenGLDriver.h

```diff
@@ -111,13 +111,13 @@ namespace video
             GLboolean stencilTestEnable = 0;
             GLboolean multisampleEnable = 1;
             struct SClipCtrl {
-                GLenum origin;
-                GLenum depth;
+                GLenum origin = GL_LOWER_LEFT;

```

Comment:
engine default is Vulkan default.

Reply:
its not engine default, its openGL default

Reply:
ok, but where are you setting our defaults?

Reply:
same place as it was before, COpenGLDriver::initDriver() or something like this

Reply:
resolved.